### PR TITLE
Clinical-development page

### PR DESCRIPTION
**Pull request by VladimirZaets**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix [#https://github.com/hlxsites/merative/102](https://github.com/hlxsites/merative/issues/102)
[https://github.com/hlxsites/merative/issues/99] (https://github.com/hlxsites/merative/issues/99)

Test URLs:
- Before: https://main--merative2--hlxsites.hlx.page/drafts/vzaets/clinical-development
- After: https://clinical-development--merative2--hlxsites.hlx.page/drafts/vzaets/clinical-development
- Origin: https://www.merative.com/clinical-development

- Before: https://main--merative2--hlxsites.hlx.page/drafts/vzaets/content-band-image
- After: https://clinical-development--merative2--hlxsites.hlx.page/drafts/vzaets/content-band-image

### CODE REVIEW

**Comment by amol-anand** [comment]
@VladimirZaets The issue you linked to #102 deals with Accordian. There is no Clinical Development issue as such. But that being said, the columns component should be generic and each type of section should not have its own CSS. It should be a generic section CSS that should cover the different types of content showing up.

File: styles/styles.css [hunk 1]
```diff
@@ -735,6 +768,38 @@ main .divider {
     border: 1px solid var(--neutral-sand);
 }
 
+main .development-time-section .columns>div>div {
+  margin-bottom: 32px
+}
+
+main .what-is-zelta-section .default-content-wrapper>p:last-child {
```
Comment (amol-anand) [inline styles/styles.css:775]: Instead of calling it the what-is-zelta-section, it can just be called something more generic like "2-column-layout" or "colorful-border" so that it can be reused across pages and is not tied to the name of the content.

Also for this piece, the display has image on the left and content on the right, but in the word document it is the other way round. The word document can have image in the first column and content in the second to mimic what is on the page. then you shouldn't have the need for more CSS

Goal is to reuse the CSS across all other pages that have the same structure/look/feel.

File: styles/styles.css [hunk 2]
```diff
@@ -735,6 +768,38 @@ main .divider {
...
+
+main .section.what-is-zelta-section .default-content-wrapper>p {
+  margin: 24px 0;
+  font-size: 16px;
+  letter-spacing: .01em;
+  line-height: 160%;
+  font-weight: 300;
+}
+
+main .accelerate-section .default-content-wrapper:last-child p {
```
Comment (amol-anand) [inline styles/styles.css:791]: Does this only relate to accelerate section? Or for all sections? Maybe this section can have a name of "no-margin" instead?

File: styles/styles.css [hunk 3]
```diff
@@ -735,6 +768,38 @@ main .divider {
...
+  margin-bottom: 0;
+}
+
+main .section.ready-to-talk-section div:nth-child(1) > p {
```
Comment (amol-anand) [inline styles/styles.css:795]: Why is this only related to ready-to-talk section? Shouldn't all sections have a bottom padding / margin? If not, then let's keep the name generic so it can be used for any content

File: styles/styles.css [hunk 4]
```diff
@@ -735,6 +768,38 @@ main .divider {
...
+  margin-bottom: 36px;
+}
+
+main .accelerate-section .columns>div>div {
```
Comment (amol-anand) [inline styles/styles.css:799]: This should be more generic for all columns in a section rather than only accelerate-section? Is that not the case?

File: styles/styles.css [hunk 5]
```diff
@@ -779,3 +844,356 @@ main .section.leadership-content .columns > div > div > h6 {
 }
 
 
+main .zelta-section {
```
Comment (amol-anand) [inline styles/styles.css:847]: class name should not be this tied to the content in this case

File: styles/styles.css [hunk 6]
```diff
@@ -779,3 +844,356 @@ main .section.leadership-content .columns > div > div > h6 {
...
+  padding-bottom: 0;
+}
+
+main .zelta-section::after {
```
Comment (amol-anand) [inline styles/styles.css:851]: class name should not be this tied to the content in this case

Comment (amol-anand) [reply styles/styles.css:851]: In this case its a border line basically so "colorful-border" or "gradient-border" might be more appropriate

File: styles/styles.css [hunk 7]
```diff
@@ -779,3 +844,356 @@ main .section.leadership-content .columns > div > div > h6 {
...
+  content: "";
+  z-index: 1;
+  background-image: var(--gradient-left-right);
+  display: block;
+  height: 1px;
+  margin-top: 32px;
+}
+
+main .section.accelerate-section .columns img {
```
Comment (amol-anand) [inline styles/styles.css:860]: should images in all sections in columns behave this way or just for this section?

**Review by amol-anand — CHANGES_REQUESTED** [review]

**Comment by VladimirZaets** [comment]
@amol-anand I updated PR and implement all styles for columns and sections for https://github.com/hlxsites/merative/issues/99 issue

**Review by amol-anand — APPROVED** [review]
